apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -869,23 +869,25 @@ When set to true, invalid utf8 strings in for example client ID, topic name, etc
 
     mqtt_max_awaiting_rel {
```

Comment:
s/mqtt_max_awaiting_rel/mqtt_max_awaiting_pubrel/

Reply:
This Label is used for `emqx_schema.erl` to reference the description of this option.
It corresponds to the configuration `mqtt.max_awating_rel` so I suggest here we should keep the same name as the configuration?

Reply:
Yes that is what I mean, we should change config name.

Reply:
that would break compatibility with older versions. I don't think it should be done in this PR or 5.0.x.
Maybe we can change it on 5.1.0 if we really need it?